src/examples/login/Login.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import { action } from "@storybook/addon-actions";
+import type { Meta, StoryObj } from "@storybook/react";
+import { Button } from "../../components/Button/Button";
+import { Checkbox } from "../../components/Checkbox/Checkbox";
+import { Heading } from "../../components/Heading/Heading";
+import { Link } from "../../components/Link/Link";
+
+export default {
+  parameters: {
+    layout: "centered",
+  },
+} satisfies Meta;
+
+export const Default: StoryObj = {
+  render: () => {
+    return (
+      <form
+        onSubmit={(event) => {
+          event.preventDefault();
+          action("on-submit")(event);
```

Comment:
Actions 애드온의 좋은 활용 사례이네요! 💯
자바스크립트 답게 lettter casing만 일관성있게 맞춰주시면 좋을 것 같습니다.

```suggestion
          action("onSubmit")(event);
```

Reply:
넹 바꿔둘게요~!!

Reply:
@DaleSeo @sounmind letter case을 위 suggestion대로 camelCase로 수정하여 commit push했습니다 :)
![image](https://github.com/user-attachments/assets/0932ea44-5c53-4a4b-9607-39fc40f82ea6)